devicetypes/smartthings/smartsense-motion-sensor.src/smartsense-motion-sensor.groovy

```diff
@@ -237,3 +239,16 @@ def configure() {
 	// battery minReport 30 seconds, maxReportTime 6 hrs by default
 	return refresh() + zigbee.batteryConfig() + zigbee.temperatureConfig(30, 300) // send refresh cmds as part of config
 }
+
+private isFirmwareVerLessThan(major, minor, build) {
+	def deviceFwVer = "${device.getFirmwareVersion()}"
+	def tempTok = deviceFwVer.tokenize('.') // We expect the format ###.###.### where ### is some integer
+	def devMajor = tempTok[0] as int
+	def devMinor = tempTok[1] as int
+	def devBuild = tempTok[2] as int
+	//def (devMajor as int, devMinor as int, devBuild as int) = deviceFwVer.tokenize('.') // Would be cleaner, but we don't seem to support multi-assignments
```

Comment:
This should also be removed before merging

Reply:
Will do. If you know why multiple assignment throws an exception for me (if it is supposed to be supported) then I'll use that.